content/practices/pre-ipm/index.md

```diff
@@ -0,0 +1,84 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+
+title: "Pre-IPM"
+linkTitle: "Pre-IPM"
+description: "An optional planning session to make the [Iteration Planning Meeting (IPM)](/practices/ipm) more productive if needed."
```

Comment:
Unfortunately we have a bug that does not allow these relative links to work properly. For now, please remove the link and we will add an Issue.

Reply:
These are resolved in commit `041a95719f6b4e022e25dfbf7eef552c741be132`, and I took the liberty of opening issue #1072 to describe it (apologies if it's a duplicate).